src/Parser/TypeParser.php

```diff
@@ -44,7 +46,9 @@ public function parse(TokenIterator $tokens): Ast\Type\TypeNode
 	private function parseAtomic(TokenIterator $tokens): Ast\Type\TypeNode
 	{
 		if ($tokens->tryConsumeTokenType(Lexer::TOKEN_OPEN_PARENTHESES)) {
+			$tokens->tryConsumeTokenType(Lexer::TOKEN_PHPDOC_EOL);
```

Comment:
This surely has some repercussions for some other parsed types. Can you add some tests demonstrating that?

Reply:
Sure. I could also provide a separate PR for this change?